plugins/search/package.json

```diff
@@ -40,8 +41,8 @@
     "@testing-library/user-event": "^12.0.7",
     "@types/jest": "^26.0.7",
     "@types/node": "^12.0.0",
-    "msw": "^0.21.2",
-    "cross-fetch": "^3.0.6"
+    "cross-fetch": "^3.0.6",
+    "msw": "^0.21.2"
   },
   "files": [
```

Comment:
I executed `yarn add qs` and the file was sorted

Reply:
We don't seem to be particularly conscious of vendor JS size right now, but I wonder if in this case, it's easy enough to avoid the `qs` dependency with a good old fashioned `encodeURIComponent()`

Reply:
myeah, qs is used in a bunch of other packages though as well - only thing then is that sometimes it's better to copy over the actual version from other packages so they all depend on the same one (yarn add may add a duplicate), but no biggie

Reply:
Yeah we already using it in other places. Just verified: we always use version 6.9.4 👍

Reply:
I have to correct myself: the yarn lock even uses three versions, but these duplicates are not originated in our direct dependencies.